package.json

```diff
@@ -14,8 +14,7 @@
     "lint.fix": "yarn lint --fix",
     "build": "ttsc --build ./packages/tsconfig.build.json",
     "posttest": "yarn lint",
-    "test": "jest --maxWorkers=2",
-    "test.watch": "yarn test --watchAll",
+    "test": "jest",
```

Comment:
Why did we lose `test.watch`? Also did you measure that it is faster without `maxWorkers`? What is the rationale?

Reply:
1. nobody was using it and you can still use It and do `yarn test --watchAll`
2. On a MacBook Jest runs with 4 workers. This value was previously set to 2 because of CircleCI: because of some virtualisation quirks jest detects 8 workers but in reality there are only two available. I've moved such limit in the CI command as you can see.

Reply:
All make sense, cool